webpack/webpack.config.js

```diff
@@ -11,11 +11,13 @@ function requireAll(array) {
 
 module.exports = {
   entry: "./src/js/index.js",
+  devtool: "inline-source-map",
```

Comment:
I think you can remove that, the source maps are set on the extending webpack configs (prod and dev)

Reply:
Will try to fix it.

Reply:
Removing it should be sufficient 👍

Reply:
done. It looks like the source maps still work